prisma/schema.prisma

```diff
@@ -15,3 +15,45 @@ model sessions {
   expires DateTime
   session Json?
 }
+
+model users {
+  id String @id @default(auto()) @map("_id") @db.ObjectId
+  uid String @unique
+  access String[] @default([])
+  bad_pwd_count Int?
+  blocked Boolean?
+  company String?
+  department String?
+  display_name String
+  force_reset_pw Boolean @default(false)
+  icon String?
+  last_logoff Int
+  last_logon Int
+  ldap Boolean?
+  lockout_time Int?
+  mail String?
+  member_of String?
+  password String
+  realms String[] @default([])
+  role String @unique
+  title String?
+  preferences userPreferences
+}
+
+
+type userPreferences {
+  complete siteComplete[]
+  star siteStarred[]
+  sort Int? @default(0)
+  config String?
+}
+
+type siteComplete {
```

Comment:
what do you think about a type name `completedSite` (and `starredSite` below)?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241